src/utils.tsx

```diff
@@ -6,4 +11,461 @@ async function callApi(url: string) {
   return body;
 }
 
-export { callApi };
+/** Contains some utility functions for handling notebook cells */
+namespace cell_utils {
```

Comment:
is this not a module we could import from somewhere? Do we need the whole thing in our code? Also if we're going to grab large sections of someone elses code, give a link to the source in a comment.

Reply:
I'll put authorship information? What format should I use to give myself/CDAT team credit?